README.md

```diff
@@ -90,10 +90,24 @@ And the following primitives for static graph optimizations:
 | Module Tracing | `s.trace(leaves, flatten)` |
 | Pattern matching | `s.find(regex_or_pattern_fn)` |
 | Operator fusion | `s[op].fuse(compiler, subgraph)` |
+| Layer decomposition | `s[op].decompose()` |
 | Partial module replacement | `s[op].replace(new_module, subgraph)` |
 | Partial gradient checkpointing | `s[op].checkpoint(subgraph)` |
-| Pipeline parallelism | `s[op].pipeline_split()` |
+| Pipeline parallelism | `s[op].cut_pipeline_stage()` |
 
+You can look for all supported primitvies with the following API:
+
+```python
+import slapo
+print(slapo.list_primitives())
+```
+
+You could also check the description of each primitive on the fly:
+
+```python
+import slapo
+print(slapo.desc_primitive("shard"))
```

Comment:
The name of this API is weird. Is this different from the `help` function? I'm not sure whether we need a separate API for this.

Reply:
I don't recall that we have `help` for now? I'm open with its name so let me know if you have any suggestion.

Reply:
I mean python's builtin `help` function. Users can easily use `help(sch.primitive)` to get the description of the primitive if they want.

Reply:
I see. Since now we register primitives on the fly, you cannot do that without initializing a schedule object; otherwise you have to `help(slapo.primitives.tensor_parallel.ShardPrimitive)`, which must be tedious. This API is more like a syntax sugar.